### ISSUE TITLE
Identify Which Hunter has Pack On

### ISSUE THREAD

**Issue by amiyuy** [TorelTwiddler/AntiDaze#2]
Add name of hunter who has Pack on to raid warnings.

**Options**
- Print hunter's name to chat
- Whisper hunter to tell them to turn off Pack

**Comment by amiyuy** [comment]
http://wowprogramming.com/docs/api/SendAddonMessage

**Comment by TorelTwiddler** [comment]
Version 6.1.0.1
Name of hunter added when the caster is someone other than the player.

Also added options for displaying in Raid Warnings or the Chat box.

**Comment by amiyuy** [comment]
Still missing: Whisper hunter to tell them to turn off Pack

**Comment by TorelTwiddler** [comment]
#1 must be completed before this, or it will spam the whisper person.

**Comment by TorelTwiddler** [comment]
#1 is fixed. Closing this and making a new issue with the whisper functionality.
